offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/DependencyManager.groovy

```diff
@@ -502,6 +513,38 @@ class DependencyManager {
             throw e
         }
 
+        CreateProductView createProductView
+        try{
+            createProductView = new CreateProductView()
```

Comment:
Very Nice just a reminder to keep the order of PR merging in mind since this constructer will be outdated in #393

Reply:
Yes I know, good point.. do you think I should remove it and also the other views? And introduce them with the respective PRs?

Reply:
Very Good!